packages/oslo-converter-uml-ea/lib/converter-handlers/ElementConverterHandler.ts

```diff
@@ -374,6 +380,23 @@ export class ElementConverterHandler extends ConverterHandler<EaElement> {
 
         switch (object.type) {
           case ElementType.Enumeration:
+            quads.push(
+              this.df.quad(
+                objectInternalId,
+                ns.rdf('type'),
+                ns.skos('Concept'),
+              ),
+            );
+            // Push Enums also as a class to keep things backward compatible.
+            // If we push Enums only as ConceptScheme, it might break some behaviour
```

Comment:
Maybe make an issue about this so if we deprecate this at some point, we know that this backwards-compatibility thing is in here.